ripple

```diff
@@ -30,13 +30,63 @@ function print_help() {
     echo "ripple run 10.0.0.1"
 }
 
+function get_default_extension() {
+    case "$(uname -s)" in
+        Darwin)
+            echo "dylib"
+            ;;
+        *)
+            echo "so"
```

Comment:
Add Windows

Reply:
Done in https://github.com/rdkcentral/Ripple/pull/187/commits/7db403949b62cedf3c850505ac925b583682d3e6